### PR TITLE
added structure an styles for top nav

### PR DESCRIPTION
**Pull request by smh7210**

### CODE REVIEW

File: assets/css/styles.css [hunk 1]
```diff
@@ -71,3 +70,62 @@ h4 {
...
+}
+
+.follow {
+  text-transform: uppercase;
+}
+
+.top-nav {
+  display: flex;
+  justify-content: center;
+  font-weight: 200;
```
Comment (RoccoSangellino) [inline assets/css/styles.css:94]: You have font-weight twice

File: assets/css/styles.css [hunk 2]
```diff
@@ -22,11 +22,10 @@ li {
 
 li a {
   color: #fff;
```
Comment (RoccoSangellino) [inline assets/css/styles.css:24]: this will apply to all list items on the site. Was this intentional?

File: assets/css/styles.css [hunk 3]
```diff
@@ -71,3 +70,62 @@ h4 {
...
+  gap: 20px;
+}
+
+.social-links li a {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.right {
```
Comment (RoccoSangellino) [inline assets/css/styles.css]: I would probably be a but more specific on what you want this class to be. "right" would be a global style

File: index.html [hunk 4]
```diff
@@ -5,13 +5,105 @@
...
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                width="24"
+                height="24"
+                viewBox="0 0 24 24"
+                style="fill: rgb(255, 215, 27)"
+              >
+                <path d="M4 6h16v2H4zm0 5h16v2H4zm0 5h16v2H4z"></path>
+              </svg>
+              <span class="phone"
```
Comment (RoccoSangellino) [inline index.html]: should both of these of the class `phone`?

**Review by RoccoSangellino — COMMENTED** [review]
Think through the styles a bit more and think about what the classes mean.

File: assets/css/styles.css (hunk 2 above)
Comment (smh7210) [reply assets/css/styles.css:24]: yes. there is only one list that i noticed wasn't that color.

File: index.html (hunk 4 above)
Comment (smh7210) [reply index.html]: I do that so that way I don't have to write .phone a to style the link

Comment (smh7210) [reply index.html]: i decided to change it a little bit

**Review by RoccoSangellino — APPROVED** [review]
Good work! Ready to be merged

**Pull request merged**